### PR TITLE
Obsolete disco

### PR DESCRIPTION
**Pull request by angakkuit-team**

### CODE REVIEW

File: asiaq.egg-info/SOURCES.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ setup.cfg
 setup.py
```
Comment (mpavlov) [inline asiaq.egg-info/SOURCES.txt:6]: `asiaq.egg-info/*` should not be tracked in git, as per the gitignore

File: reports/nosetests.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-nose.config: INFO: Set working dir to /Users/ejen/pydevels/asiaq
```
Comment (mpavlov) [inline reports/nosetests.txt:1]: `reports/*` should not be tracked

**Comment by mpavlov** [comment]
Is this intended to be merged back to https://github.com/amplifylitco/asiaq? Why not just use a normal branch or an official fork? This seems like it was copied file by file.

**Comment by angakkuit-team** [comment]
This repo is more likely for me to experiment with some ideas i am having after hearing from some people who looked at asiaq and were confused about how to use it.

So it is not an official fork from repo. But more like a start from taking existing asiaq and start to bastardize/make asiaq attractive to them

**Comment by mpavlov** [comment]
This is a bad idea.

It's already not trivial to maintain and update Asiaq to keep up with AWS changes and new services. If we create a pseudo-fork the pains will be multiplied rather than halved. Progress will be significantly reduced because of fracturing the community interested in advancing the tool.

IMO, the better path is to submit this PR against Asiaq. It benefits everyone and it can be done in a backwards-compatible way so as to not disrupt existing automation.

**Comment by angakkuit-team** [comment]
well said.

but when the patch is accepted. all existing configurations will be broken unless they are updated.

But it can make work easier.

**Comment by eugenejen** [comment]
@mpavlov : you actually pointed out an important problem in previous comment that all software tools built on top of AWS APIs. AWS is also moving fast. So any team adopts asiaq has to suffer from waiting for new AWS features to be added into Asiaq.

I can see the reluctance from my current team to use asiaq, they are more willing to write their own directly against AWS APIs themselves.

**Comment by mpavlov** [comment]
FWIW, with a little work, this PR doesn't have to break existing configuration.

And I would posit that if Asiaq, in its current form, is sufficiently far from your needs that it requires a _copy_ (rather than an eventually-mergeable fork) then it really is better to start from scratch and write your own AWS API wrappers.

**Comment by eugenejen** [comment]
In fact this seems a good idea. Instead of mocking around the existing tools. Maybe just create new tools directly building on top of AWS APIs. Then let those tools to be part of asiaq that coexists with existing tools.

Since each tools is also a contract to its users. According to microservice philosophy, we should honor the contract and create new tools as new contracts for users.

Then we should adhere to following the original unix philosophy, such that keeping every tool specific in purpose and do one thing well.

**Comment by eugenejen** [comment]
this pr will be obsolete. I will rebase the repo and use asiaq as upstream. then I will change all 'disco' to 'asiaq' and submit the PR back to amplifylitco

**Pull request closed without merging**